src/main/java/pl/byczazagroda/trackexpensesappbackend/exception/WalletNotSavedException.java

```diff
@@ -0,0 +1,8 @@
+package pl.byczazagroda.trackexpensesappbackend.exception;
+
+public class WalletNotSavedException extends AppRuntimeException {
+
+    public WalletNotSavedException() {
+        super("Sorry. Something went wrong and your Wallet is not saved. Contact administrator.");
```

Comment:
This message should be different. We are not Talking with user but with frontend. Frontend should send message to user. Are they getting message from our rest api or they will create owns messages?